src/Servers/HttpSys/src/WebHostBuilderHttpSysExtensions.cs

```diff
@@ -9,6 +9,9 @@
 
 namespace Microsoft.AspNetCore.Hosting
 {
+    /// <summary>
+    /// Provides extensions method to use Http.sys as the server for the web host.
```

Comment:
A little lower down the docs use `HttpSys`. I like `Http.sys` better, but could you update the docs on the `UseHttpSys` method as well to be consistent?